### PR TITLE
full uppercase const

### PR DESCRIPTION
**Pull request by ArtificialOwl**
fix last character of `StorageConfig::MOUNT_TYPE_PERSONAl`

### CODE REVIEW

**Review by nickvergessen — APPROVED** [review]

File: apps/files_external/lib/Lib/StorageConfig.php [hunk 1]
```diff
@@ -37,7 +37,7 @@
  */
 class StorageConfig implements \JsonSerializable {
 	public const MOUNT_TYPE_ADMIN = 1;
-	public const MOUNT_TYPE_PERSONAl = 2;
```
Comment (icewind1991) [inline apps/files_external/lib/Lib/StorageConfig.php:40]: Have we checked if this is references in any other app?

Might be worth to leave the old const in place for compatibility

Comment (nickvergessen) [reply apps/files_external/lib/Lib/StorageConfig.php:40]: Hmm I thought PHP was case forgiving and it just errors in Psalm. But seems that is only for Class names:
https://3v4l.org/bboIB

So yeah, should keep the old one in parallel.

File: apps/files_external/lib/Service/DBConfigService.php [hunk 2]
```diff
@@ -37,7 +37,7 @@
  */
 class DBConfigService {
 	public const MOUNT_TYPE_ADMIN = 1;
-	public const MOUNT_TYPE_PERSONAl = 2;
```
Comment (nickvergessen) [inline apps/files_external/lib/Service/DBConfigService.php:40]: same here

Comment (ArtificialOwl) [reply apps/files_external/lib/Service/DBConfigService.php:40]: @icewind1991 should we merge or we keep both (`DBConfigService` and `StorageConfig`) definition ?

File: apps/files_external/lib/Lib/StorageConfig.php (hunk 1 above)
Comment (ArtificialOwl) [reply apps/files_external/lib/Lib/StorageConfig.php:40]: now marked as ~~deprecated~~

**Review by icewind1991 — APPROVED** [review]

**Review by joshtrichards — APPROVED** [review]

**Pull request merged**